projects/angular/src/utils/variables/API.md

```diff
@@ -0,0 +1,20 @@
+# Global properties-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-global-borderradius | It changes the radius of the elemnt's corner and is applied globally.
+| --clr-global-borderwidth | It changes the width of the border and is used globally.
+| --clr-global-app-background | It changes the background colour of the body and is used globally.
+| --clr-global-selection-color | It changes the background colour when the element is selected and is used globally.
+| --clr-global-on-selection-color | It changes the background colour when the element is selected and is used globally.
+| --clr-global-on-selection-color | When an element is chosen, it changes the colour of the text inside the element and is used globally.
+| --clr-global-hover-color | On mouse over, it changes the background colour of an element.
+| --clr-global-content-header-font-color | It changes the colour of the text for all the headings and is used globally.
+| --clr-global-font-color | It modifies the text colour of the element's description and is used globally.
+| --clr-global-success-color | On success, it sets the foreground colour value of an element's text.
+| --clr-global-error-color | On error, it sets the foreground colour value of an element's text.
+| --clr-close-color--normal | It sets the colour of the close icon.
+| --clr-close-color--normal-opacity | sets the level of transparency for an element.
+| --clr-close-color--hover | It sets the colour of the close icon when the mouse cursor is over it
+| --clr-close-color--hover-opacity | sets how transparent an element is when the mouse is over it.
+| --clr-popover-box-shadow-color | It changes one or more shadows on an element and used globally.
```

Comment:
It sets the color of the highlight of the close icon

Reply:
Updated ,As suggested